app/services/prompt_augmentor.rb

```diff
@@ -11,22 +11,46 @@ def augment
   end
 
   def prompt
-    @prompt ||= if @search_hits.any?
-      prompt = "Here is some context that may help you answer the following question:\n\n"
-      @search_hits.each do |hit|
-        prompt << "#{hit.content}\n\n"
-      end
+    if @search_hits.none?
+      return nil
+    elsif @search_hits.none?(&:relevant)
```

Comment:
Are you seeing search hits that aren't relevant returned?

In a chat scenario, you shouldn't be, and V1::ChatController doesn't allow the client to specify `include_irrelevant`.

Reply:
I think I was just seeing empty(none?) search hits and unnecessarily augmented prompt in the ui.